<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Server_VSClass</class>
 <widget class="QMainWindow" name="Server_VSClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>768</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>业务服务平台软件</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>241</width>
      <height>641</height>
     </rect>
    </property>
    <property name="title">
     <string>业务种类列表</string>
    </property>
    <widget class="QPushButton" name="DeleteBtn">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>610</y>
       <width>41</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>-</string>
     </property>
    </widget>
    <widget class="QPushButton" name="RunBtn">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>610</y>
       <width>41</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>+</string>
     </property>
    </widget>
    <widget class="QTableWidget" name="ServerList">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>221</width>
       <height>581</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>10</y>
      <width>1011</width>
      <height>641</height>
     </rect>
    </property>
    <property name="title">
     <string>设备列表</string>
    </property>
    <widget class="QTableWidget" name="clientList">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>991</width>
       <height>611</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>660</y>
      <width>1271</width>
      <height>51</height>
     </rect>
    </property>
    <property name="title">
     <string>状态栏</string>
    </property>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>54</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>运行状态</string>
     </property>
    </widget>
    <widget class="QLabel" name="StatusLabel">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>20</y>
       <width>500</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>运行正常</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1280</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>功能</string>
    </property>
    <addaction name="actionDMTM"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>查看日志</string>
    </property>
    <addaction name="action_SYSLog"/>
    <addaction name="action_DataLog"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_2"/>
  </widget>
  <action name="actionDMTM">
   <property name="text">
    <string>补抄数据</string>
   </property>
   <property name="toolTip">
    <string>补抄数据</string>
   </property>
  </action>
  <action name="action1">
   <property name="text">
    <string>1</string>
   </property>
  </action>
  <action name="action_SYSLog">
   <property name="text">
    <string>系统日志</string>
   </property>
  </action>
  <action name="action_DataLog">
   <property name="text">
    <string>数据日志</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
 <slots>
  <slot>on_RunBtn_clicked()</slot>
  <slot>On_SetTimeBtn_Clicked()</slot>
  <slot>on_clientList_customContextMenuRequested(QPoint)</slot>
  <slot>qwe()</slot>
 </slots>
</ui>
